data/events/scripts/monster.lua

```diff
@@ -33,3 +33,7 @@ function Monster:onDropLoot(corpse)
 		end
 	end
 end
+
+function Monster:onSpawn(position, startup, artificial)
+   return true
```

Comment:
Spaces insted of tabs

Reply:
Over my dead body.

Reply:
Well not really, you misunderstood me :) 
You got spaces there, I want you to replace them with a tab.